crates/provider/src/native.rs

```diff
@@ -153,9 +144,9 @@ impl<T: FileSystem + Send + Sync> Provider for NativeProvider<T> {
 
     async fn spawn_temp(
         &self,
-        node: Node,
-        files_inject: Vec<FileMap>,
-        files_get: Vec<FileMap>,
+        _node: Node,
+        _files_inject: Vec<FileMap>,
+        _files_get: Vec<FileMap>,
```

Comment:
What is the goal of this input ?

Reply:
Doesn't really matter for this PR. Once implementation is done we can address the name based on usability